### ISSUE TITLE
Import Lakhi Jungle Khalsa

### ISSUE THREAD

**Issue by Sarabveer** [ShabadOS/database#1339]
seems to be missed when importing Sarabloh Granth

**Comment by Sarabveer** [comment]
![image](https://user-images.githubusercontent.com/4297171/80844734-6959a900-8bd5-11ea-8fdf-976a69b30a89.png)

**Comment by Sarabveer** [comment]
@Sarigpani please check if this is in Sarabloh Granth

**Comment by Sarigpani** [comment]
No, this is from Dasam Granth Asfokat Kabit, however not found in all current standardised saroops. 

![image](https://user-images.githubusercontent.com/60820452/80864954-040dc280-8c7e-11ea-9c01-33363996f6c1.png)

**Comment by Sarigpani** [comment]
@Sarabveer 
![image](https://user-images.githubusercontent.com/60820452/80865592-6f599380-8c82-11ea-9bf5-7be979922511.png)
![image](https://user-images.githubusercontent.com/60820452/80865624-91ebac80-8c82-11ea-9b20-8c068688fe03.png)

**Comment by Sarabveer** [comment]
There are many paath bheds, will go with Amrit Keertan. Will need to figure out which composition to add this too.

I am considering renaming Sarabloh Granth as we only have a few shabads in there.

**Comment by Sarabveer** [comment]
<img width="658" alt="image" src="https://user-images.githubusercontent.com/4297171/166608784-ebc0e04f-3cd7-47dc-bcb4-d9ac8175b211.png">
